packages/elements/src/calendar/index.ts

```diff
@@ -800,7 +801,29 @@ export class Calendar extends ControlElement implements MultiValue {
   }
 
   /**
-   * Run when tap event happens ot table.
+   * get parent of node that contain value
+   * @param event Keyboard event
+   * @returns html element
+   */
+  private getCellParent(event: KeyboardEvent): HTMLElement | null {
+    const path = event.composedPath();
+    for (let i = 0; i <= path.length; i += 1) {
+      const node = path[i] as HTMLElement;
+      if (node === this) {
+        return null;
+      }
+      if (node.nodeType !== Node.ELEMENT_NODE) {
+        continue;
+      }
+      if (node.getAttribute('role') === 'button' && node.part.value.includes('cell-content')) {
```

Comment:
I have a bit concern about these query attribute and part. for the worse case user could send slot with these condition `role=button` `part=cell-content`, should we mention it some where. or am just overthinking 🧐

Reply:
I think the user won't send role=button and part=cell-content with slot.